Add contributing guide back

jvican opened the pull request:
I've written up a quick contributing guide to get started in the bloop
codebase. It's not perfect but it's definitely better than having
nothing... I hope new contributors feel more comfortable working on the
codebase after reading the codebase.

I'm happy to improve its contents and the format, so feedback is very
much welcome.

Fixes https://github.com/scalacenter/bloop/issues/846 and https://github.com/scalacenter/bloop/issues/208

Review by @tkroman and @torkelrogstad

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 0 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

torkelrogstad commented on docs/contributing-guide.md:
```diff
@@ -0,0 +1,168 @@
+## Learn the developer workflow
+
+The recommended way to build and hack on bloop is by using bloop. Run
+`bloopInstall` if you are not using Metals to export the build to the
```
```suggestion
`bloopInstall` in your sbt shell if you are not using Metals to export the build to the
```

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

torkelrogstad commented on docs/contributing-guide.md:
```diff
@@ -0,0 +1,168 @@
+### Depend on SNAPSHOT releases
+
+If you are using bloop as a library dependency, you can add Scala Center
+Bintray's resolver and use the version right away.
```
Would be nice to either give the steps for that, or point to somewhere that does

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

torkelrogstad commented on docs/contributing-guide.md:
```diff
@@ -0,0 +1,168 @@
+If you have installed bloop via the CURL script in the website, you can use a
+local Python script to override your installation in `$HOME/.bloop`:
+
+1. Give executable permission to `$BLOOP_REPO/frontend/target/install.py`.
```
```suggestion
1. Give executable permission to the install script: `chmod +x $BLOOP_REPO/frontend/target/install.py`.
```

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 0 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 0 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 0 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 0 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
Performance test finished successfully.

Benchmarks is based on merging with master

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

jvican commented:
Wow... GitHub and our benchmark infrastructure has gone completely crazy. If you're a casual bystander, ignore the messages created by @scalacenter/bloop-team

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 1 running.

bloopoid commented:
performance test scheduled: 1 job(s) in queue, 0 running.

jvican commented on website/versioned_docs/version-1.2.5/contributing-guide.md:
```diff
@@ -0,0 +1,169 @@
+1. `config` contains Scala 2.10 and 2.12 module to read and write configuration files.
+1. `docs` and `docs-gen` defines our docs infrastructure.
+1. `benchmark-bridge` and `benchmarks` defines our compiler benchmark infrastructure.
+1. `launcher` is an application to spawn a bloop server and establish a
```
The launcher was merged a while ago in latest master and released in 1.2.5 (or maybe before?) so you should be able to see two files named `launcher.json` and `launcher-test.json` under `.bloop` :smile:

jvican commented:
Thank you for your feedback @tkroman @torkelrogstad That was useful :+1: I've just addressed the feedback. I'm merging, I hope to be improving this guide iteratively as more and more doubts start to show up in our Gitter channel room. Feel also free to contribute sections to it for workflows that are not obvious.
